src/main/java/org/isf/utils/jobjects/LocalDateSupportingJDateChooser.java

```diff
@@ -0,0 +1,57 @@
+package org.isf.utils.jobjects;
+
+import com.toedter.calendar.IDateEditor;
```

Comment:
Import order is incorrect; the order is `static` imports, `java`, `javax`, `org`, `com` and then anything else.